index.html

```diff
@@ -272,134 +315,188 @@ <h2>Glossary</h2>
+converted to ä in this form. This form most closely matches legacy 
+usage. The formal definition is D120 in <a href="http://www.unicode.org/versions/latest/ch03.pdf#G49537">
+Section 3.11, Normalization Forms</a>.</q> See also [=normalization=].</p>
+
+<p><dfn class="lint-ignore export" data-lt="unicode normalization form d|nfd">Normalization Form D</dfn> (<strong>NFD</strong>). <a href="https://www.unicode.org/glossary/#normalization_form_c">Unicode definition</a>: <q cite="https://www.unicode.org/glossary/#normalization_form_d">A normalization form that erases any canonical differences, and produces a <strong>decomposed</strong> result. For example, ä is converted to a + umlaut in this form. This form is most often used in internal processing, such as in collation. The formal definition is D118 in <a href="http://www.unicode.org/versions/latest/ch03.pdf#G49537"> Section 3.11, Normalization Forms</a>.</q> See also [=normalization=].</p>
 
 
 
 
 <div class="letter_anchor" id="p">P</div>
-	
-<p><dfn class="lint-ignore export">Producer</dfn>. When talking about strings on the Web, the W3C Internationalization group refers to a producer as any process where natural language string data is created for later storage, processing, or interchange.</p>
 
-<p><dfn class="lint-ignore export">Paragraph direction</dfn>. The initial <a>base direction</a>  of  a paragraph or string, which resolves to either <em>left-to-right</em> or <em>right-to-left</em>. Nested embedding controls may be used to change the direction of an inline range of text, but the paragraph direction sets the starting point which the <a>Unicode Bidirectional Algorithm</a> uses  to calculate the directions of the embedded levels. For more details, see <a href="http://www.unicode.org/reports/tr9/">Unicode Standard Annex #9, Unicode Bidirectional Algorithm</a> [[UAX9]], especially definitions BD2–BD5.<br><span class="see-also">See also the definition in the <a href="https://www.unicode.org/glossary/#paragraph_direction">Unicode Standard</a>.</span></p>
+<p><dfn class="lint-ignore export">Paragraph direction</dfn>. The initial <a>base direction</a> of  a paragraph or string, which resolves to either <em>left-to-right</em> or <em>right-to-left</em>. Nested embedding controls may be used to change the direction of an inline range of text, but the paragraph direction sets the starting point which the <a>Unicode Bidirectional Algorithm</a> uses  to calculate the directions of the embedded levels. For more details, see <a href="http://www.unicode.org/reports/tr9/">Unicode Standard Annex #9, Unicode Bidirectional Algorithm</a> [[UAX9]], especially definitions BD2–BD5.<br><span class="see-also">See also the definition in the <a href="https://www.unicode.org/glossary/#paragraph_direction">Unicode Standard</a>.</span></p>
```

Comment:
link `left-to-right` and `right-to-left`, since they are terms (vs. `em` here)

Reply:
Need to think about whether the terms we're defining are right-to-left (adjectival) or 'right to left'.  See a recent discussion and some guidelines on this topic from the Unicode Editorial committee.